lib/attackMonitoring.js

```diff
@@ -152,6 +155,16 @@ let enableAttackMonitoring = function (policyJSON, callbackFunction) {
             const allowedModuleArray = policyJSON.outBoundRequest.allowedDomains.filter(domain => isPureObject(domain));
             const reportUri = new URL(policyJSON.reportUri);
 
+            // Set the violationLimit
+            if (policy.hasOwnProperty('maxViolationsPerMinute')){
+                violationLimit = policy['maxViolationsPerMinute'];
+            } else {
+                violationLimit = 100;
```

Comment:
This else condition can be removed and the default value of the violationLimit can be set during the variable declaration as mentioned in earlier comment.